code/world.dm

```diff
@@ -12,6 +12,14 @@
 
 var/savefile/panicfile
 /world/New()
+	var/extools_path = system_type == MS_WINDOWS ? "byond-extools.dll" : "libbyond-extools.so"
+	if(fexists(extools_path))
+		call(extools_path, "maptick_initialize")()
+	// lib missing, show warning if on linux
+	else if(system_type == UNIX)
+		// extools on linux does not exist and is not in the repository as of yet		
+		warning("Dream Daemon appears to be running on Unix, but there is no extools library included with this repository. Performance may differ significantly from a Windows host. This warning will not show if libbyond-extools.so is added to the root of the game directory.")
```

Comment:
Probably worth dropping the `if` in case it's ever missing/deleted
```suggestion
	else
		// extools on linux does not exist and is not in the repository as of yet
		warning("There is no extools library for this system included with this build. Performance may differ significantly than if it were present. This warning will not show if [extools_path] is added to the root of the game directory.")
```